diff --git a/dist/pull.js b/dist/pull.js
index 9b397bd2e63588dc76c57cd6b752e6363af4df62..166f1bab7a9c7b5d2021b5e72c232ff7397ec8c6 100644
--- a/dist/pull.js
+++ b/dist/pull.js
@@ -183,7 +183,7 @@ function writePage(page, finalMarkdown) {
     ++counts.output_normally;
 }
 const notionLimiter = new limiter_1.RateLimiter({
-    tokensPerInterval: 3,
+    tokensPerInterval: 1,
     interval: "second",
 });
 let notionClient;
diff --git a/dist/transform.js b/dist/transform.js
index 2cf0164846478c97a7a06f3a7a125e148b34776b..752ee1814b6857a4b816267c8cb05c9607e26544 100644
--- a/dist/transform.js
+++ b/dist/transform.js
@@ -196,6 +196,10 @@ function getFrontMatter(page) {
     frontmatter += `slug: ${(_a = page.slug) !== null && _a !== void 0 ? _a : ""}\n`;
     if (page.keywords)
         frontmatter += `keywords: [${page.keywords}]\n`;
+    const dateProperty = page.getDateProperty("Date", "", true) || page.getDateProperty("PublishDate", "", true) || "";
+    if (dateProperty) {
+        frontmatter += `date: ${dateProperty}\n`;
+    }
     frontmatter += "---\n";
     return frontmatter;
 }
